src/1A5830.c

```diff
@@ -72,10 +72,10 @@ void play_hit_sound(Actor* actor, f32 x, f32 y, f32 z, u32 hitSound) {
                     sfx_play_sound_at_position(SOUND_E2, 0, x, y, z);
                     break;
                 case ACTOR_PARTNER:
-                    sfx_play_sound_at_position(SOUND_EA, 0, x, y, z);
+                    sfx_play_sound_at_position(SOUND_FIRE, 0, x, y, z);
```

Comment:
These sound names don't really mean anything to me. The sound of fire in what context?

Reply:
They play accordingly to the dealt damage type "element"

Reply:
then I think maybe FIRE_DAMAGE and SOUND_ELECTRIC or w/e would be better. Or even SOUND_ELEM_FIRE and SOUND_ELEM_ELECTRIC. just to make it clear that these are battle sounds and not just the sound of fire or electricity in some general sense. if the other one wasn't electricity, my b. I forgot what it was and this tiny diff isn't showing me